### PR TITLE
feat: improve Otello page

### PR DESCRIPTION
**Pull request by astagi**
This PR adds Otello embedded video and fixes a link to the docs

## Checklist

* [x] I followed the indications in [CONTRIBUTING.md](https://github.com/italia/developers.italia.it/blob/master/CONTRIBUTING.md)
* [ ] The documentation related to the proposed change has been updated accordingly (also comments in code).
* [ ] Have you written new tests for your core changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?
* [x] Ready for review! :rocket:

### CODE REVIEW

**Comment by bfabio** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>There is a problem with the commit message
> fix: Otello docs link
- subject must not be sentence-case, start-case, pascal-case, upper-case</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 7aef899bc1be323217efdfc119247a6d03b8ab9d
</p>

File: _platforms/it/otello.md [hunk 1]
```diff
@@ -14,7 +14,9 @@ layout: platform
 
 ## Intro
 
-OTELLO è la piattaforma dell'[Agenzia delle Dogane e dei Monopoli](https://www.agenziadoganemonopoli.gov.it/) che gestisce il processo che consente ai soggetti extra-UE di ottenere lo sgravio diretto o il rimborso dell'IVA sui beni acquistati sul territorio italiano.
+OTELLO è la piattaforma dell'[Agenzia delle Dogane e dei Monopoli](https://www.agenziadoganemonopoli.gov.it/){:target="_blank"} che gestisce il processo che consente ai soggetti extra-UE di ottenere lo sgravio diretto o il rimborso dell'IVA sui beni acquistati sul territorio italiano.
+
+<iframe title="Otello Tax free shopping – Italiano" width="560" height="315" src="https://www.youtube.com/embed/ejVjgkFUPxo" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" class="d-block mx-auto mb-2" allowfullscreen></iframe>
```
Comment (libremente) [inline _platforms/it/otello.md]: For privacy policy matters I think it's better to just add a link to the page where the video stays (if there's an embed somewhere on the official pages even better). Also, directly embedding a video can result in accessibility issues. WDYT?

**Comment by developers-italia-droid** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>There is a problem with the commit message
> fix: Otello video is responsive now
- subject must not be sentence-case, start-case, pascal-case, upper-case</td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  There is a problem with the commit message
> fix: Otello external links have target="_blank"
- subject must not be sentence-case, start-case, pascal-case, upper-case
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>There is a problem with the commit message
> fix: Otello docs link
- subject must not be sentence-case, start-case, pascal-case, upper-case</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against d48e4c43c5d47df586d059dd2b39693cdf643658
</p>

File: _platforms/it/otello.md (hunk 1 above)
Comment (astagi) [reply _platforms/it/otello.md]: @libremente fixed with an internal image

**Review by libremente — APPROVED** [review]

**Pull request merged**